test/mail_with_ldap.bats

```diff
@@ -40,10 +40,8 @@ function setup_file() {
 		-e REPORT_RECIPIENT=1 \
 		-e ENABLE_SASLAUTHD=1 \
 		-e SASLAUTHD_MECHANISMS=ldap \
-		-e SASLAUTHD_LDAP_SERVER=ldap \
-		-e SASLAUTHD_LDAP_BIND_DN=cn=admin,dc=localhost,dc=localdomain \
-		-e SASLAUTHD_LDAP_PASSWORD=admin \
-		-e SASLAUTHD_LDAP_SEARCH_BASE=ou=people,dc=localhost,dc=localdomain \
+		-e SASLAUTHD_LDAP_SERVER=ldap://ldap \
+		-e SASLAUTHD_LDAP_SSL=0 \
```

Comment:
If this is being removed elsewhere, why is it added here now? The associated commit message wasn't very clear.

Presumably for testing the case in `setup-stack.sh`?

Reply:
`SASLAUTHD_LDAP_SSL=0` together with the protocol being specified would previously break, with this I basically check if it's ignored in favor of the protocol in `SASLAUTHD_LDAP_SERVER`.

Reply:
If that's being kept there should be a comment about it for context when another contributor runs across it.

Seems like it's being asked to drop from `setup-stack.sh` though, so perhaps has no relevance here anymore either?

Reply:
This should be removed as well, see my comment below :)